clients/cobol-lsp-vscode-extension/src/test/suite/testHelper.ts

```diff
@@ -73,6 +77,11 @@ export async function closeAllEditors() {
   });
 }
 
+export async function undo() {
+  await vscode.commands.executeCommand("undo");
+  await sleep(100);
```

Comment:
Should undo function be inlined? No one uses it outside the module and it is sensitive to execution delays, so it shouldn't be used like this outside closeActiveEditor logic.